projects/canopy/src/lib/card/docs/promotions/guide.stories.mdx

```diff
@@ -0,0 +1,513 @@
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+</div>
+```
+## Additional development details
+
+### LgCardComponent
+
+The main body of the promotion card. It's composed by the following components:
+
+- `LgCardHeroImageComponent`
+- `LgCardContent`
+
+#### Inputs
+The orientation input arranges the direct children of the card in vertical or horizontal layout, a responsive way.
+It accepts an object, where the **key** is a mobile-first breakpoint from the standard list of available breakpoints: `xs`, `sm`, `md`, `lg`, `xl`, `xxl`.
+The value is from the set of available Orientation options.
+The current available options for Orientation are `vertical`, `horizontal`, `reverse-vertical`, `reverse-horizontal`.
+
+| Name           | Description                                                     | Type                    | Default | Required |
+|----------------|-----------------------------------------------------------------|-------------------------|---------|----------|
+|   `variant`    | The variant of the card: `promotion`                            | `string`                |`default`| Yes      |
+| `orientation`  | The orientation of the card content: `{ sm: 'vertical' }`       | `OrientationResponsive` |   n/a   |  No      |
+
+### Responsive examples of orientations
+
+Let's say that at the `sm` breakpoint you want to apply `vertical` orientation, then at the `md` breakpoint apply `horizontal` orientation, then at the `lg` breakpoint apply `vertical` orientation.
+The object you'd pass into `orientation` would look like this:
+
+```html
+
+<lg-card [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }"></lg-card>
+
+```
+
+### LgCardHeroImageComponent
+
+This is the primary layout section of the card component, it is used to contain a hero Image or a Brand Icon.
+An image is added via either the `coverImageUrl` input, or 'imageSrc' input(see below for differences). A Brand Icon is added via content projection using the `<lg-brand-icon>` component.
```

Comment:
I think `imageSrc` should be enclosed in backticks like `coverImageUrl`

Reply:
we now have only ``src`` as per the instructions bellow :)